scripts/lib.sh

```diff
@@ -8,17 +8,19 @@ function mine_blocks() {
 }
 
 function open_channel() {
-    LN_ADDR="$($FM_LN1 newaddr | jq -e -r '.bech32')"
+    # check that both nodes are synced
+    await_lightning_node_block_processing
+
+    LN_ADDR="$($FM_LIGHTNING_CLI newaddr | jq -e -r '.bech32')"
     $FM_BTC_CLIENT sendtoaddress $LN_ADDR 1
     mine_blocks 10
-    FM_LN2_PUB_KEY="$($FM_LN2 getinfo | jq -e -r '.id')"
-    export FM_LN2_PUB_KEY
-    FM_LN1_PUB_KEY="$($FM_LN1 getinfo | jq -e -r '.id')"
-    export FM_LN1_PUB_KEY
-    $FM_LN1 connect $FM_LN2_PUB_KEY@127.0.0.1:9001
-    until $FM_LN1 -k fundchannel id=$FM_LN2_PUB_KEY amount=0.1btc push_msat=5000000000; do sleep $POLL_INTERVAL; done
+    $FM_LNCLI getinfo
```

Comment:
This `getinfo` without processing its output seems unnecessary/left from debugging something?

Reply:
Good catch. I'm going to merge this (since it's approved) and then make a follow-up PR to remove this.

Reply:
https://github.com/fedimint/fedimint/pull/1856